pkg/resources/account_role.go

```diff
@@ -17,8 +16,9 @@ import (
 
 var accountRoleSchema = map[string]*schema.Schema{
 	"name": {
-		Type:     schema.TypeString,
-		Required: true,
+		Type:             schema.TypeString,
+		Required:         true,
+		DiffSuppressFunc: suppressIdentifierQuoting,
 		// TODO(SNOW-999049): Uncomment once better identifier validation will be implemented
```

Comment:
this still points to the identifiers rework epic and we have already a follow-up for this validation. Will all the deprecated TODO links be replaced in this or separate change?

Reply:
I'll create a separate pr to point every SNOW-999049 todo to the right ticket nr.